agents/kubevirt/fence_kubevirt.py

```diff
@@ -0,0 +1,126 @@
+#!@PYTHON@ -tt
+
+import os
+import sys, re
+import logging
+import atexit
+sys.path.append("@FENCEAGENTSLIBDIR@")
+from fencing import *
+from fencing import fail, fail_usage, run_delay, EC_STATUS, SyslogLibHandler
+
+try:
+    from kubernetes.client.exceptions import ApiException
+except ImportError:
+    logging.error("Couldn\'t import kubernetes.client.exceptions.ApiException - not found or not accessible")
```

Comment:
These needs to be pass, so we can build it without having the libs installed:
https://ci.kronosnet.org/job/fence-agents-build-all-voting/721/fence-agents-build-all-voting=centos-8-x86-64/console

Reply:
This error log doesn't cause to failure, no?
The failure reason in the error is this line, no?
`07:49:34 diff: /srv/workspace/fence-agents-build-all-voting/fence-agents-build-all-voting/centos-8-x86-64/tests/data/metadata/fence_kubevirt.xml: No such file or directory`
How should I add `tests/data/metadata/fence_kubevirt.xml` file?

Reply:
Run `make xml-upload` and add tests/data/metadata/fence_kubevirt.xml to the commit.

Reply:
It's both:
13:49:34 ERROR:root:Couldn't import kubernetes.client.exceptions.ApiException - not found or not accessible